src/style/properties.js

```diff
@@ -597,9 +597,9 @@ class CrossFadedProperty<T> implements Property<T, ?CrossFaded<T>> {
         } else {
             assert(!value.isDataDriven());
             return this._calculate(
-                value.expression.evaluate({zoom: parameters.zoom - 1.0}),
-                value.expression.evaluate({zoom: parameters.zoom}),
-                value.expression.evaluate({zoom: parameters.zoom + 1.0}),
+                value.expression.evaluate(new EvaluationParameters(parameters.zoom - 1.0, parameters)),
+                value.expression.evaluate(parameters),
+                value.expression.evaluate(new EvaluationParameters(parameters.zoom + 1.0, parameters)),
```

Comment:
The second argument to `EvaluationParameters` here (and two lines above) isn't strictly necessary, because expression evaluation doesn't make use of any properties of `EvaluationParameters` other than `zoom`.

Reply:
Then again, if this fact were to change in the future, omitting the argument would cause a bug wherein whatever new important property would get stripped.  👍 LGTM